date.php

```diff
@@ -80,7 +80,7 @@
             <?php
             if ($accessLevel >= 2) {
                 echo '
-                    <a class="button" href="addEvent.php?date=' . $date . '">
+                    <a class="button" href="pointsProgForm.php?date=' . $date . '">
```

Comment:
This commit changed instances of addEvent.php to pointsProgForm.php in a few different files to essentially fully replace addEvent with pointsProgForm. It may have been better to comment out addEvent instead of replacing it, in case something breaks or we need to know where it was in the future.

Reply:
I agree with your suggestion of commenting out addEvent for future reference. It may be a good idea, before too many additions are made, to clarify what code was replaced.